src/js/tabs/sensors.js

```diff
@@ -242,6 +242,10 @@ TABS.sensors.initialize = function (callback) {
             ConfigStorage.set({'graphs_enabled': checkboxes});
         });
 
+        let altitudeHint_e = $('.tab-sensors .wrapper.altitude .plot_control .helpicon');
```

Comment:
5 different classes? How about `.tab-sensors #helpAltitudeHint` and then slap `id="helpAltitudeHint"` in the html?

Reply:
That's all classes that are actually there and used.
But I get your message, classes are bad. Changed.